synapse/lib/storm.py

```diff
@@ -1824,6 +1825,8 @@ class Runtime(s_base.Base):
     opaque object which is called through, but not dereferenced.
 
     '''
+
+    _admin_reason = s_hiveauth._allowedReason(True, isadmin=True)
```

Comment:
could we just put this in s_hiveauth?

Reply:
We could but we'll rip it out in the not distant future.

Reply:
yeah good point